README.md

```diff
@@ -419,7 +419,7 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | DISCO | New York City, NY; or Austin, TX | Software Engineer Intern @ [New York City, NY](https://www.csdisco.com/careers/software-engineer-intern-c), [Austin, TX](https://www.csdisco.com/careers/software-engineer-intern-6) required to provide proof of eligibility to work in the United States without restrictions|
 | [Snackpass](https://boards.greenhouse.io/snackpasspt/jobs/4658696004) | San Francisco, CA / Remote | Internship - Software Engineering |
 | [Tesla](https://www.tesla.com/careers/search/job/software-engineering-internship-summer-2023-153316) | Palo Alto, California | Software Engineering Internship |
-| [Spectrum](https://jobs.spectrum.com/job/greenwood-village/2023-summer-intern-software-engineer/4673/37073255520) | Multiple Locations | Software Engineering Intern|
+| [Spectrum](https://jobs.spectrum.com/job/charlotte/2023-summer-intern-information-technology/4673/36113961104) | Multiple Locations | Software Engineering Intern|
```

Comment:
The SWE Internship offering is still available from my side, at least.

Are you trying to add the IT internship?